Agents/APIAgent/data_classes/iris.py

```diff
@@ -9,7 +9,7 @@
 XSD = "http://www.w3.org/2001/XMLSchema#"
 TIME = "http://www.w3.org/2006/time#"
 TS = "https://www.theworldavatar.com/kg/ontotimeseries/"
-RML4TS = "http://www.theworldavatar.com/ontology/rml4ts/rml4ts.owl#"
+RML4TS = "https://www.theworldavatar.com/kg/rml4ts.owl#"
```

Comment:
You seem to have missed two occurrences of "4"...

Reply:
This has been addressed in the latest change commit. The name is unified to 'rml4ts', as 'RML for Timeseries', I errorneously put 'rmlts' in the file names and READMES.